conf/ocsci/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# This is the default configuration file which will be merged with file passed
+# by:
+#
+# * --ocsci-conf file.yaml parameter.
+# * --cluster-conf cluster.yaml parameter - this will be rendered under
+#   ENV_DATA section
+#
+# Then it will be rendered with jinja2 to be able to reuse vars defined in
+# different sections. The sections which are with capital letters will be
+# available directly in ocsci.config module.
+#
+# Please when you reuse some variable follow this convention:
+# ceph_image: '{{ DEFAULTS["CEPH_IMAGE"] }}'
+# Order of the quotas is important! Start with apostrophe like in example!
+# ------------------------------------------------------------------------
+# This is the place for common default variables used in multiple places.
+# Values from this section are then available from tests from ocs.defaults
+# module.
+DEFAULTS:
+  AWS_REGION: "us-east-2"
+  ROOK_CLUSTER_NAMESPACE: "openshift-storage"
+  KUBECONFIG_LOCATION: "auth/kubeconfig"  # relative from cluster_dir
+  API_VERSION: "v1"
+  CEPH_IMAGE: "quay.io/rhceph-dev/rhceph-4.0-rhel-8"
+  ROOK_IMAGE: "quay.io/rhceph-dev/rook"
+  DEPLOYMENT_PLATFORM: "AWS"
+
+
+
+# in this RUN section we will keep default parameters for run of OCS-CI
+RUN:
+  log_dir: "/tmp"
+  run_id: null  # this will be redefined in the execution
+  kubeconfig_location: '{{ DEFAULTS["KUBECONFIG_LOCATION"] }}'
+  cli_params: {}  # this will be filled with CLI parameters data
+
+# In this section we are storing all deployment related configuration
+DEPLOYMENT:
+  installer_version: "4.1.0-rc.3"
+
+# Section for reporting configuration
+REPORTING:
+  email:
+    address: "ocs-ci@redhat.com"
```

Comment:
Is this address ok?

Isn't there a chance that it can generate a lot of spam from all users that try to run tests in their local environment and won't change this address? Also is this address ok here in upstream?

Reply:
Currently in pytest we don't have this emailing enabled at all.
Once we implement it, it would be with some --email parameter enabled (By default I wouldn't send any email as don't see the sense of it for local run when you see the results in your console)

Just thinking in advance and wanted to put something under this reporting section as example ;)

Reply:
If this is just an example, I would rather make it explicit by using `example.com` domain.